src/status_im2/contexts/chat/composer/actions/view.cljs

```diff
@@ -58,14 +59,14 @@
     {:icon                true
      :size                32
      :accessibility-label :send-message-button
-     :on-press            #(send-message props state animations window-height)}
+     :on-press            #(send-message props state animations window-height edit)}
     :i/arrow-up]])
 
 (defn send-button
-  [props {:keys [text-value] :as state} animations window-height images?]
+  [props {:keys [text-value] :as state} animations window-height images? edit]
   (let [btn-opacity (reanimated/use-shared-value 0)
         z-index     (reagent/atom (if (and (empty? @text-value) (not images?)) 0 1))]
-    [:f> f-send-button props state animations window-height images? btn-opacity z-index]))
+    [:f> f-send-button props state animations window-height images? btn-opacity z-index edit]))
```

Comment:
I'd say it's too many arguments at this point, probably better to move them to a map instead